src/tests/rust_fmt_argument_max_padding.rs

```diff
@@ -0,0 +1,244 @@
+            "%%\n%5555555%55555555555u555"
+        ),
+        (
+            MockTime::new(-21761, 255, 58, 0, 37, 37, 620822272, 255, 43690, 50396945, false, 3, ""),
+            "%R%S%00000400000R%RR%%%\u{3}\0%\0\u{1b}\u{1b}\u{1b}"
+        ),
+        (
+            MockTime::new(13566191, 1, 0, 34, 35, 37, 623190565, 0, 8741, 2457877026632491810, true, 606414171, ""),
+            "\0\0\0\0\0\0\0\"%2536361%172049423024124\"2\"\"9\"%^^^YN^\""
+        ),
+        (
+            MockTime::new(1749024784, 96, 0, 96, 104, 96, 620796776, 0, 15968, 0, false, 6307376, ""),
+            "%u\0\n\0\0\u{10}\0`\u{31b}@\0\0\x002>`\0%33333333s33333#33333333333333333333333333333333333\u{13}33333333333u\0\0\0\0\0%u\u{4}\0\0\0\u{10}\0`"
+        ),
+    ];
+
+    for (time, format) in test_cases {
+        let err = get_format_err(&time, format);
+        assert!(matches!(err, Error::WriteZero));
+    }
+}
+
+#[test]
+#[rustfmt::skip]
+fn test_rust_136932_reduce_fmt_argument_width_and_precision_fuzzer_failures_bytes() {
+    let test_cases: [(MockTime<'_>, &[u8]); 8] = [
+        (
+            MockTime::new(674049317, 244, 42, 40, 180, 106, 4096628178, 1, 0, 3239384020795663872, false, 1734567725, ""),
+            b"\xd6\x00\x00\x00\x00\xda\xc7\xda%\xc1B%-666666F\xff\xff\xff"
+        ),
+        (
+            MockTime::new(167837442, 64, 255, 10, 96, 255, 4294901760, 255, 65535, -1, true, 235864006, ""),
+            b"\x88%%%%%%%%%%%%%777777%%%\x0e"
+        ),
+        (
+            MockTime::new(1375731722, 82, 82, 82, 85, 82, 1392508927, 82, 20992, -48935437123235246, false, -1145324613, ""),
+            b"\xbb\xbb%\x00\x00RR%\xbb\xbb\x00\x00\x00B%00124261%R\xff\x00\xbb\xbb\xbb\xbb\x00\x00\x00\x00\x00RRR\xa6RR"
+        ),
+        (
+            MockTime::new(0, 0, 0, 0, 92, 0, 704643072, 0, 242, 5548434740937171968, false, 0, ""),
+            b"\x00\x00%7777773%"
+        ),
+        (
+            MockTime::new(458557, 0, 64, 128, 128, 128, 4278222976, 255, 65535, -205, true, 65535, "\0\0\0"),
+            b"\x00%00324248X14\xff"
+        ),
+        (
+            MockTime::new(-774831746, 209, 209, 209, 13, 13, 218959117, 13, 3341, 0, false, 218955776, ":\r\r"),
+            b"\xff\x00\xff\xff\x08%%%%%%%%%%%%%5%%%%3333333%%"
+        ),
+        (
+            MockTime::new(130826, 3, 223, 29, 37, 37, 623191333, 93, 13861, 2892759177136963579, true, 623191333, "!I\u{1}\0\u{3}"),
+            b"\xdf\x1d%\x1d?\xff\xdb\xfe#\x15I%$\x00\x00\x00%]%86995%%6\xfb"
+        ),
+        (
+            MockTime::new(37454602, 0, 0, 0, 0, 9, 1048576, 255, 65535, -1, false, 255, "\0\0\0\0\0/\0"),
+            b"y%999999%q)%%z"
+        ),
+    ];
+
+    for (time, format) in test_cases {
+        let err = get_format_err_bytes(&time, format);
+        assert!(matches!(err, Error::WriteZero));
+    }
+}
+
+#[test]
+#[cfg(feature = "alloc")]
+fn test_format_specifiers_large_width_success() {
```

Comment:
We should also do the same tests for the precision, which suffers from the same regression.

Reply:
After checking we don't use the precision specifier anywhere in the code.

Reply:
sounds good. thoughts on how to fix this? I think probably we'll want to write padding in `u16::MAX`-sized chunks up until we can delegate to the formatting machinery?